flytekit/image_spec/image_spec.py

```diff
@@ -51,12 +52,20 @@ class ImageSpec:
         commands: Command to run during the building process
         tag_format: Custom string format for image tag. The ImageSpec hash passed in as `spec_hash`. For example,
             to add a "dev" suffix to the image tag, set `tag_format="{spec_hash}-dev"`
+        copy: This option allows the user to specify which source files to copy from the local host, into the image.
+            Not setting this option means to use the default flytekit behavior. The default behavior is:
+                - if fast register is used, source files are not copied into the image (because they're already copied
+                  into the fast register tar layer).
+                - if fast register is not used, then the LOADED_MODULES (aka 'auto') option is used to copy loaded
+                  Python files into the image.
+
+            If the option is set by the user, then that option is of course used.
     """
 
     name: str = "flytekit"
     python_version: str = None  # Use default python in the base image if None.
     builder: Optional[str] = None
-    source_root: Optional[str] = None
+    source_root: Optional[str] = None  # a.txt:auto
```

Comment:
`a.txt:auto`. is there a plan to support this?
it looks nicer honestly

Reply:
separate pr?